packages/core/src/types/IViewport.ts

```diff
@@ -18,7 +18,15 @@ export interface ViewReferenceSpecifier {
    * and cannot be shared across different view types such as stacks and
    * volumes, or two viewports showing different orientations or slab thicknesses.
    */
-  sliceIndex?: number | [number, number];
+  sliceIndex?: number;
+  /**
+   * The end index - this requires sliceIndex to be specified.
+   */
+  sliceRangeEnd?: number;
```

Comment:
does it makes sense to have

```
sliceIndex?: number -> for single index
sliceRange?: [number, number] -> for range
```

Reply:
No, I'm trying to get away from sliceIndex at all since it really isn't deterministic between different display sets contain the same set of images.  I would have liked to get rid of it in this PR, but that was more of an API change than I was willing to contemplate here, whereas the simple change to have two different values instead of number | [number,number] is much more internally consistent with the current API, even though it is a small change.